scripts/changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
@levithomason I completely agree with making this an npm module (this is what ShellJS is for, anyway). I actually ready wrote out a release.js script (translated from your release.sh). If that's fine with you, I can publish that under the ShellJS org (shouldn't be an issue, since it's a derivative work under the MIT license).